docs/src/modules/components/AppLayoutDocs.js

```diff
@@ -120,23 +120,27 @@ export default function AppLayoutDocs(props) {
 
   const { canonicalAs } = pathnameToLanguage(router.asPath);
   let productName = 'MUI';
+  let teamName = 'Core';
   if (canonicalAs.startsWith('/material-ui/')) {
     productName = 'Material UI';
   } else if (canonicalAs.startsWith('/base-ui/')) {
     productName = 'Base UI';
   } else if (canonicalAs.startsWith('/x/')) {
     productName = 'MUI X';
+    teamName = 'X';
   } else if (canonicalAs.startsWith('/system/')) {
```

Comment:
It would be great to drop this logic based on URL and to use `productId` & `productCategoryId` & `convertProductIdToName` instead.